### ISSUE TITLE
start.sh : change ping target in testing internet connection

### ISSUE THREAD

**Issue by alexxtasi** [ttn-zh/ic880a-gateway#57]
Hi all
About the case that in the beginning of ```start.sh``` there is an internet connection testing, by pining to google.com ...
- there are network environments where net admins do filter icmp traffic to google (maybe not in the field, but while testing gateways in laboratories)
- there is a workaround to this problem by ralphtheninja's pull request [here](https://github.com/ttn-zh/ic880a-gateway/pull/18) and reixd 's suggestion

wouldn't be more generic to test gateway's connectivity by pinging to a more TTN related url, and what would that be ? 
I believe this should be less possible to be filtered and if no ping to it, then the gateway is facing a real connectivity problem !!

regards

**Comment by gonzalocasas** [comment]
@alexxtasi I agree with changing icmp checks to something else, but changing google's endpoint with ttn might not be that meaningful, chances of google being down are lower than ttn's website being down.

A different way of checking instead of icmp would be something like:
```
if nc -zw1 google.com 443; then
  # all is good, ready to start concentrator
fi
```
but right now I don't have the time to test it and put it on the installer, if you do, please send a PR.

cheers
